lemur/auth/views.bak

```diff
@@ -0,0 +1,720 @@
+"""
```

Comment:
This whole file looks like it's unneeded.

Reply:
oops - my bad. tx.
Gone now.